hw/ip/edn/doc/_index.md

```diff
@@ -132,7 +132,7 @@ The required `generate` and `reseed` commands must also be custom generated by f
 These `generate` commands will be issued as necessary to meet the bandwidth requirements of the endpoints.
 The `reseed` commands will be issued once every `MAX_NUM_REQS_BETWEEN_RESEEDS` generate requests.
 For details on the options for application interface commands please see the [CSRNG IP Documentation]({{< relref "hw/ip/csrng/doc/_index.md" >}}).
-Once the CSRNG instance has been instantiated, and the `generate` and `reseed` commands have been loaded, auto request mode can be entered by programming the {{< regref "CTRL" >}} register with `EDN_ENABLE`, `AUTO_REQ_MODE` and `BOOT_REQ_MODE` all asserted.
+Once the CSRNG instance has been instantiated, and the `generate` and `reseed` commands have been loaded, auto request mode can be entered by programming the {{< regref "CTRL" >}} register with `EDN_ENABLE` and `AUTO_REQ_MODE` fiedls are enabled.
```

Comment:
nit: fiedls -> fields

Reply:
@mwbranstad just one question (and I'm a bit late on this), since `auto_req_mode` can only be set after the interface has been instantiated, does that mean the software needs to wait until the instantiation is complete..? Or is it just more of programming one after the other? The description in the hjson made it sound like it was a just a mode to set, but this statement makes it sound like there is a timing dependency.

Reply:
the protocol for both edn and csrng software ports is that a cmd request is placed, then poll the cmd_sts rdy bit before doing the next command, or enabling the auto_req_mode. the description has been changed to gear toward an expected use case (INS, GEN, RES command sequence), but the software port is really generic (any command can be placed there) and in all cases the cmd rdy bit should be polled.